### DISCUSSION TITLE
v1.5.0-beta

### DISCUSSION THREAD

**Discussion by fl0ppy-d1sk** [bunkerity/bunkerweb#464] [Help]
Documentation : https://docs.bunkerweb.io/1.5.0-beta/

Docker tags : 
- `bunkerity/bunkerweb:1.5.0-beta`
- `bunkerity/bunkerweb-scheduler:1.5.0-beta`
- `bunkerity/bunkerweb-autoconf:1.5.0-beta`
- `bunkerity/bunkerweb-ui:1.5.0-beta`

Linux packages : https://packagecloud.io/app/bunkerity/bunkerweb/search?q=1.5.0-beta&filter=all&dist=

Changelog : 
- Refactoring of almost all the components of the project
- Dedicated scheduler service to manage jobs and configuration
- Store configuration in a database backend
- Improved web UI and make it working with all integrations
- Improved internal LUA code
- Add Redis support when using clustered integrations
- Add RHEL integration
- Add Vagrant integration
- Improved CI/CD

<hr /><em>This discussion was created from the release <a href='https://github.com/bunkerity/bunkerweb/releases/tag/v1.5.0-beta'>v1.5.0-beta</a>.</em>

**Comment by TheophileDiot** [comment]
:rocket:

**Comment by lesensei** [comment]
Just wanted to leave a note saying I've been using the 1.5 beta at home for a couple of days now, and it's been working nicely. The web UI in an autoconf setup is a nice touch, that helped me realize parts of my config were wrong. Thanks for your work !
